### DISCUSSION TITLE
Windows Terminal Preview v1.15.228

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#13768] [Announcements]
This is another servicing release for the Preview channel of Windows Terminal! We fixed that Alt+Tab issue!

> **Note**
> People in the Beta channel of the Windows Insider program will receive _1.15_ as a _Stable_ channel update while we test out coming features for the next version of Windows. It is roughly equivalent to the build included here, but it does not include the experimental text rendering engine.

It contains the following other things as well:

### Usability
* We've restored the ability for <kbd>Alt+Tab</kbd> to restore the Terminal after it was minimized with the taskbar icon (#13624)
* Terminal will no longer replace colored backgrounds with blank spaces on first launch (#13665)
* We will once again display underlines, hyperlinks, and more to the end of the line instead of getting tired and stopping early (#13661)
* Sessions handed off from the Windows Console will no longer stick around with an ominous and annoying "process exited with code ..." message, unless you explicitly configure them to.
   * To accomplish this, we've changed the default value of `closeOnExit` to a new value, `automatic`. Automatic close-on-exit determines whether
   to close based on whether the process exited gracefully *and* whether the process was spawned by Terminal. (#13560) (#13649)
* Select All and Mark Mode will now trigger scrolling to make sure that one of the selection endpoints is visible. (#13660)
* `SendInput` with high unicode characters will no longer fail (#13667)
* Text input in Japanese, Vietnamese, Korean and Chinese should be more reasonably switch between alphanumeric modes (#13678) (#13677)

### Reliability
* We've upgraded to XAML 2.7.3 to fix a crash in closing the Settings page (#13761)
* The "Open Terminal Here" context menu item should show up more reliably (and crash less) (reverted PR #13206)
* We've solved--or at least, reduced the incidence of--one source of deadlocks in rendering (#13758)
* We'll try much harder to defibrillate a Terminal session that can't talk to the "primary" Terminal session to improve reliability (#13604)
* We will now listen to signals the OS sends us telling us that it's taking us down for an update. It's not going to help us stop it form happening,
  but it puts us in a better position to handle it later (#13614)

### Performance
* We're preparing to fix an issue with jump list generation that results in a slower-than-expected launch (but we aren't there yet) (#13688)
* There is now only one tab color picker in the world, and all ~~users~~ terminal tabs now have to share it (to improve performance) (#13736 

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.15.2282.0'>Windows Terminal Preview v1.15.228</a>.</em>

**Comment by PhMajerus** [comment]
Is there something blocking the preview version from the MS Store, or is the update supposed to be live there?
The release version got updated yesterday, but the preview version is still at 1.15.2003.0, seeking apps updates does not help.
Testing from Windows 11 Release Preview channel.

Update: 7th day, update still not visible in the MS Store. 😢

Good news, on the end of the 7th day, the Store finally offered the update. 🥳

**Reply by DHowett** [reply]
Just the inherent slowness in my pressing of the button to make it live for people. We had a scare (#13774) that made me wait a couple extra days to publish the releases. Thanks for asking :)
